Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.11.6-alpine3.18
+LABEL maintainer="lev2099@gmail.com"
+
+ENV PYTHONNBUFFERED 1
```

Comment:
There is a typo in the environment variable name `PYTHONNBUFFERED`. It should be `PYTHONUNBUFFERED` to ensure Python output is sent straight to the terminal without being buffered.